### PR TITLE
golint: use golint to check package util

### PR DESCRIPTION
**Pull request by HuKeping**
part of #14756

Signed-off-by: Hu Keping hukeping@huawei.com

### CODE REVIEW

File: utils/stubs.go [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 package utils
 
+// ExperimentalBuild is only for experimental check by Jenkins by now.
```
Comment (cpuguy83) [inline utils/stubs.go]: This isn't really accurate as it is used in `docker info`.

perahps: `ExperimentalBuild is a stub which always returns false for builds that do not include the "experimental" build tag`

Comment (HuKeping) [inline utils/stubs.go]: Sure, thanks.

**Comment by duglin** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Pull request merged**